metrics/web/lib/common/presentation/metrics_theme/model/project_group_dialog_theme_data.dart

```diff
@@ -2,66 +2,43 @@ import 'package:flutter/material.dart';
 
 /// A class that stores the theme data for the project group dialogues.
 class ProjectGroupDialogThemeData {
-  /// The default padding for this theme data.
-  static const _defaultPadding = EdgeInsets.zero;
+  /// A primary [Color] for main elements of the project group dialog.
+  final Color primaryColor;
 
-  /// An empty space between the main content and dialog's edges.
-  final EdgeInsets padding;
+  /// A secondary [Color] for elements of the project group dialog.
+  final Color accentColor;
 
-  /// An empty space that surrounds the content.
-  final EdgeInsets contentPadding;
+  /// A background [Color] of the project group dialog.
+  final Color backgroundColor;
 
-  /// A [TextStyle] of the content text for the project group dialog.
-  final TextStyle contentTextStyle;
+  /// A [TextStyle] for the [TextField]s of the project group dialog.
+  final TextStyle textFieldTextStyle;
 
-  /// A [TextStyle] of the content secondary text for the project group dialog.
-  final TextStyle contentSecondaryTextStyle;
+  /// A [TextStyle] of the dialog title.
+  final TextStyle titleTextStyle;
 
   /// A border [Color] of the project group dialog content.
   final Color contentBorderColor;
 
-  /// An empty space that surrounds the actions.
-  final EdgeInsets actionsPadding;
+  /// A [TextStyle] of the content text for the project group dialog.
+  final TextStyle contentTextStyle;
+
+  /// A [TextStyle] of the content secondary text for the project group dialog.
+  final TextStyle contentSecondaryTextStyle;
```

Comment:
Can we get back on the naming (whole matching Dialog elements names to theme property names)? We can have a quick call I guess on this to sort it out. 

Also, should we split textStyles?